src/extractors/youtube-new/requester/initializer/initializer.ts

```diff
@@ -0,0 +1,26 @@
+import { YT_CONSTANTS, DEVICE_TYPE } from 'extractors/youtube-new/utils/constants';
+import { YTContext } from 'extractors/youtube-new/utils/types';
+import Config from './config';
+import Context from './context';
+
+export class Initializer {
+  private _mWebContext: YTContext;
+  private _appContext: YTContext;
+
+  async init(): Promise<void> {
+    // Request API key from YouTube's Innertube API (as well as client object)
+    const { client } = await new Config().getInnertubeConfig(YT_CONSTANTS.URL.YT_MOBILE);
+
+    // Generate context(s) for future requests
+    this._mWebContext = new Context(DEVICE_TYPE.MOBILE_WEB).getContext(client);
+    this._appContext = new Context(DEVICE_TYPE.MOBILE_APP).getContext(client);
+  }
+
+  get mWebContext(): YTContext {
```

Comment:
because `_mWebContext` and `_appContext` are private properties, they cannot be used or read within the Initializer class. I added these new getters to expose the private properties